lib/assets/stylesheets/modules/_toc.scss

```diff
@@ -113,41 +123,46 @@
         z-index: 10;
 
         &__label {
+          @include govuk-font($size: 16);
           display: block;
           position: relative;
           z-index: 10;
 
-          padding: $gutter-half $gutter-half 10px;
+          padding: govuk-spacing(3) govuk-spacing(3) 10px;
 
-          background: $grey-4;
+          background: govuk-colour("light-grey");
           @supports(backdrop-filter: blur(2px)) {
-            background: transparentize($grey-4, 0.05);
+            background: transparentize(govuk-colour("light-grey"), 0.05);
             backdrop-filter: blur(2px);
           }
-          border-bottom: 1px solid $grey-2;
+          border-bottom: 1px solid govuk-colour("mid-grey");
           box-shadow: 1px 1px 3px rgba(0, 0, 0, 0.1);
-          color: $text-colour;
+          color:  $govuk-text-colour;
           text-decoration: none;
+
+          &:focus {
+            @include govuk-focused-text;
+          }
         }
 
         &__icon {
           width: 20px;
           height: 20px;
           float: right;
 
-          background-image: file-url('govuk-icn-numbered-list.png');
+          background-image: url('/images/govuk-icn-numbered-list.png');
           background-repeat: no-repeat;
           background-position: left center;
 
-          @include device-pixel-ratio {
-            background-image: file-url('govuk-icn-numbered-list@2x.png');
+          @include govuk-device-pixel-ratio {
+            background-image: url('/images/govuk-icn-numbered-list@2x.png');
             background-size: 20px 20px;
           }
         }
       }
 
       .toc__list {
-        margin-right: $gutter * 1.5;
+        margin-right: govuk-spacing(7);
       }
 
       .toc__close {
```

Comment:
The close link is using the default user agent styles when focused:

<img width="612" alt="Screen Shot 2019-09-04 at 11 24 11" src="https://user-images.githubusercontent.com/121939/64247345-9058c180-cf06-11e9-9076-96ec7d954888.png">